Source/Mosa.Korlib/System.IO/File.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) MOSA Project. Licensed under the New BSD License.
+
+using System.Runtime.CompilerServices;
+
+namespace System.IO;
+
+public static class File
```

Comment:
is this compatible with the dotnet equivalent?

Reply:
Yup, it's just missing quite a bit of functions, but otherwise the methods added here are fully compatible with .NET.